.github/workflows/UnitTest.yml

```diff
@@ -0,0 +1,55 @@
+name: Unit test
+
+on:
+  create:
+    tags:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '20 00 1 * *'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
The default value for this is `true`.

If I get it correctly, Github Action currently doesn't support `allow_failure`, it would be nice to disable this to keep all jobs running.